a3p-integration/proposals/z:acceptance/test-lib/psm-lib.js

```diff
@@ -161,41 +190,17 @@ export const implementPsmGovParamChange = async (question, voting, io) => {
     address,
   );
   console.log('charterAcceptOfferId', charterAcceptOfferId);
-  const [brands, instances] = await Promise.all([
-    agoric
-      .follow('-lF', ':published.agoricNames.brand', '-o', 'text')
-      .then(brandsRaw =>
-        Object.fromEntries(marshaller.fromCapData(JSON.parse(brandsRaw))),
-      ),
-    agoric
-      .follow('-lF', ':published.agoricNames.instance', '-o', 'text')
-      .then(instancesRaw =>
-        Object.fromEntries(marshaller.fromCapData(JSON.parse(instancesRaw))),
-      ),
-  ]);
-  console.log('charterAcceptOfferId', charterAcceptOfferId);
+  const { brands, instances } = await snapshotAgoricNames();
```

Comment:
Let's not put any more effort into improving these functions that can be replaced by `@agoric/client-utils`.